manual/source/introduction.rst

```diff
@@ -146,14 +146,11 @@ links.
 .. index::
    link
    plotting
-   NXdata (base class); plotting
-   NXmonitor (base class); plotting
-   NXdetector (base class); plotting
    
 :ref:`Design-Links`
-    Links are used to reference the plottable data from ``NXdata``
-    when the data is provided in other groups
-    such as ``NXmonitor`` or ``NXdetector``.
+    Links are pointers to existing data somewhere else in the hierarchy. They
+    are used to have the same information be represented in different places
```

Comment:
A bit convoluted. Maybe better as "are used to represent the same information in different places".

Reply:
@PeterC-DLS Agree?